README.md

```diff
@@ -1,24 +1,98 @@
-# Duke project template
+# CS2103T Individual Project (iP)
+
+This is a project log for my individual project
```

Comment:
Great job on logging in your own progress on the README.md. Provides clear overview of milestones and helps review commits.